css/main.css

```diff
@@ -447,12 +448,15 @@ body {
 }
 
 .profession__header {
+    position: absolute;
```

Comment:
It must be done as flex. Do not use absolute for positioning such things as headers in internal blocks (blocks which located out of header and footer).